clients/benchmarks/CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ target_include_directories( hipblas-bench
 )
 
 if (NOT WIN32)
-    target_link_libraries( hipblas-bench PRIVATE hipblas_fortran_client lapack cblas stdc++fs )
+    target_link_libraries( hipblas-bench PRIVATE hipblas_fortran_client lapack cblas stdc++fs gfortran )
```

Comment:
I can't imagine that linking libgfortran is the way to fix flang.

Reply:
This change can be dropped. I've prepared updates to the build infrastructure so that all Fortran components are built with the same toolchain. In general, that is the only way to ensure that all the proper runtime libraries are linked. If you start explicitly linking runtimes like in this patch, you're embedding assumptions about the toolchain into your build.